db/migrate/20180608204308_create_tweets.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTweets < ActiveRecord::Migration[5.2]
+  def change
+    create_table :tweets do |t|
+      t.string :tweet
```

Comment:
oh i see you did the adding user to the migration in a separate migration. word.

Reply:
wait so what is the difference between the tweet and `string` columns? i feel like maybe we only need one of the two columns?

Reply:
no the string column was an accident i just noticed, i prob typed it incorrectly when i generated teh migration i'll fix that

Reply:
i think you can go ahead and just modify your original migration because it isn’t merged.
so i’d say remove the db/migrate/20180612211841_remove_string_from_tweet.rb and then modify db/migrate/20180608204308_create_tweets.rb
also maybe we should call it like tweet.body instead of tweet.tweet
just thinking of names names names
other than that this is looking tight